### PR TITLE
feat: disabled FAB

### PR DESCRIPTION
**Pull request by jaulz**
### Motivation

Allows disabling of FAB.

### Test plan

image

### CODE REVIEW

**Comment by ferrannp** [comment]
@jaulz can you check those merge conflicts?

**Comment by jaulz** [comment]
@ferrannp done. Let me know if you need something else 😊

**Comment by jaulz** [comment]
@satya164 just changed it

image

File: src/components/FAB/FAB.js [hunk 1]
```diff
@@ -81,13 +85,14 @@ class FAB extends React.Component<Props> {
...
       color: customColor,
+      disabled,
       onPress,
       theme,
       style,
       ...rest
     } = this.props;
 
-    const { backgroundColor = theme.colors.accent } =
+    const { backgroundColor = disabled ? grey500 : theme.colors.accent } =
```
Comment (ferrannp) [inline src/components/FAB/FAB.js]: Can you use same `disabled` color as `Button`: ?

```js
    if (disabled) {
        backgroundColor = color(theme.dark ? white : black)
          .alpha(0.12)
          .rgb()
          .string();
```

**Pull request closed without merging**